cmlreaders/readers/eeg.py

```diff
@@ -118,6 +123,13 @@ def read(self) -> np.ndarray:
             raise NotImplementedError("FIXME: we can only read all channels now")
 
         with h5py.File(self.filename, 'r') as hfile:
+            try:
+                self.rereferencing_possible = bool(hfile['monopolar_possible'][0])
```

Comment:
Would it make sense to raise some sort of warning if this is true so the user knows that they won't be able to reference this particular dataset?

Reply:
nvm, I see this will happen inside of as_timeseries